creativecloud/blocks/sidenav/sidenav.js

```diff
@@ -149,11 +149,19 @@ function appendResources(rootNav, resourceLink) {
   const merchTag = createTag('merch-sidenav-list', { sidenavListTitle, 'daa-ll': generateDaaLL(sidenavListTitle, 'resources') });
   merchTag.append(tag);
   const label = literals[1].trim();
+  const ariaLabel = resourceLink.getAttribute('aria-label') || '';
   const link = createTag('sp-sidenav-item', { href: resourceLink.href, 'daa-ll': generateDaaLL(sidenavListTitle, 'special-offers'), target: '_blank', selected: false });
   if (resourceLink.href && resourceLink.href.startsWith('http')) {
     link.append(document.createTextNode(label));
     const icon = createTag('sp-icon-link-out-light', { class: 'right', slot: 'icon' });
```

Comment:
@bozojovicic as per the SWC doc, setting label attribute here should be enough:

https://opensource.adobe.com/spectrum-web-components/components/icon/#accessibility

>aria-hidden is set to true by default for Icons. The label attribute suppresses this and adds the label text as the aria-label of the icon.

Reply:
with your approach, if the UI component refreshes, your dom change will be lost.